htdocs/projet/class/project.class.php

```diff
@@ -2670,10 +2670,10 @@ public function createWeeklyReport()
 					$actioncomm->email_sender = '';
 					$actioncomm->email_to = $to;
 
-					$actioncomm->errors_to = $errors_to;
+					$actioncomm->errors_to = (string) $errors_to;
```

Comment:
var $errors_to is already declared as string so cast should be useless.
i found a $errors_to = 0 line of code. This is surely the line that make the CTI saying there is a problem. I fixed it for $errors_to but other cases are probably similar...